content/kubernetes/active-active/preview/create-reaadb.md

```diff
@@ -100,13 +100,15 @@ Before creating an Active-Active database on Redis Enterprise for Kubernetes, yo
     apiVersion: app.redislabs.com/v1alpha1
     kind: RedisEnterpriseActiveActiveDatabase
     metadata:
-      name: example-aadb-1
```

Comment:
let's stay consistent with the same name as in other examples in the doc.
and keep "example-aadb-1"